### PR TITLE
[Retail] Retribution Paladin Dragonflight Partial Support

### PR DESCRIPTION
**Pull request by seagomezar**
### Description

The support of Retribution Paladin has been enabled in this commit.

- The spellBook for Retribution has been updated with the new talent three
- Crusade, ArtOfWar, ExecutionSentence, and Empyrean Power were updated.
- RighteousVeredict and SantifiedWrath were updated as well.
- The proper call to combat.hasTalent has been updated to send talent
- instead of a spell.
- Code Uncommented in the combat parser log.

This Contributes to https://github.com/WoWAnalyzer/WoWAnalyzer/issues/5775

### Motivation

I love to see the retribution analysis again in WoWAnalyzer and I am happy to contribute to open-source products.
Since this is my first contribution to the project I want to make sure I am doing the right things.

### Testing

- Test report URL:  /report/zGhTjK8APRt7a1Qp/22-4+Dathea,+Ascended/Klamuz/standard
- Screenshot(s):
Analysis
Warning

### CODE REVIEW

File: src/parser/getConfig.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export default function getConfig(
   combatant?: CombatantInfoEvent,
 ) {
   const expansionConfigs = AVAILABLE_CONFIGS.filter((config) => config.expansion === expansion);
+  console.log({ expansionConfigs });
```
Comment (trevorm4) [inline src/parser/getConfig.ts]:
```suggestion
```

File: src/interface/report/PlayerLoader.tsx [hunk 2]
```diff
@@ -286,6 +286,9 @@ const PlayerLoader = ({ children }: Props) => {
       combatant,
     );
   const build = combatant && getBuild(config, combatant);
+
+  console.log('este', config);
+
```
Comment (trevorm4) [inline src/interface/report/PlayerLoader.tsx]:
```suggestion
```

File: src/common/SPELLS/paladin.ts [hunk 3]
```diff
@@ -351,7 +391,12 @@ const spells = spellIndexableList({
     name: 'Zeal',
     icon: 'spell_holy_sealofblood',
   },
-  EXECUTION_SENTENCE_DEBUFF: {
+  //   EXECUTION_SENTENCE_DEBUFF: {
+  //     id: 343257,
+  //     name: 'Execution Sentence',
+  //     icon: 'spell_paladin_executionsentence',
+  //   },
```
Comment (trevorm4) [inline src/common/SPELLS/paladin.ts]: Remove this if it's unneeded or uncomment it

**Review by trevorm4 — CHANGES_REQUESTED** [review]

File: src/parser/getConfig.ts (hunk 1 above)
Comment (seagomezar) [reply src/parser/getConfig.ts]: Removed

File: src/interface/report/PlayerLoader.tsx (hunk 2 above)
Comment (seagomezar) [reply src/interface/report/PlayerLoader.tsx]: Removed

File: src/common/SPELLS/paladin.ts (hunk 3 above)
Comment (seagomezar) [reply src/common/SPELLS/paladin.ts]: Removed!

**Comment by seagomezar** [comment]
@trevorm4 Thank you for the review I think it is ready now.

**Review by trevorm4 — APPROVED** [review]

**Pull request merged**